client/core/core.go

```diff
@@ -137,6 +140,24 @@ func (ctx CoreContext) SignBuildBroadcast(name string, msg sdk.Msg, cdc *wire.Co
 	return ctx.BroadcastTx(txBytes)
 }
 
+func (c CoreContext) NextSequence(address []byte) (int64, error) {
```

Comment:
Function header comments please

Reply:
Header added.

Reply:
uh.. not quite hehehe - I just pushed with a comment though

Reply:
Oops, too many windows open apparently. Thanks!